guides/17_Migration_to_3_0.md

```diff
@@ -123,8 +132,60 @@ const eventReporter = new EventReporter(...)
-})
 ```
 
 ### `forEachObserver` use cases
-The `eventDidReceive` function that was part of `AudioVideoObserver` has been moved to `EventObserver` which is a observer that the `EventController` now handles. Because of this if you were you were to call `eventDidReceive` through `forEachObserver` on `AudioVideoController` this functionallity will no longer be possible in 3.x, however you will still be able to call `eventDidReceive` by using the `publishEvent` method on `EventController`. If you have a use case not covered by this method you can implement your own `EventController` or make a feature request.
+
+The `eventDidReceive` function that was part of `AudioVideoObserver` has been moved to `EventObserver` which is a observer that the `EventController` now handles. Because of this if you were you were to call `eventDidReceive` through `forEachObserver` on `AudioVideoController` this functionality will no longer be possible in 3.x, however you will still be able to call `eventDidReceive` by using the `publishEvent` method on `EventController`. If you have a use case not covered by this method you can implement your own `EventController` or make a feature request.
+
+### WebRTC Metrics
+
+Before in 2.x:
+
+The `DefaultStatsCollector` used a hybrid approach to obtain WebRTC stats from browser:
+
+* For Chromium-based browsers, call [legacy (non-promise-based) `getStats` API](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getStats#obsolete_syntax)
+* For any other browsers, call [standardized (promise-based) `getStats` API](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getStats#syntax)
+
+After in 3.x:
+
+The [legacy (non-promise-based) `getStats` API](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getStats#obsolete_syntax) will be removed and [standardized (promise-based) `getStats` API](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getStats#syntax) will be used for all browsers.
+
+SDK exposed some common WebRTC metrics publicly via the `metricsDidReceive` event. We did not make any change to `metricsDidReceive` itself. However In V3, the legacy WebRTC metric specs will be removed or replaced by equivalent standardized metrics. Here is a table that summarizes all the changes and offers the steps to migrate.
+
+| No. | MetricSpec                | SpecName                       | Change  | Migration Step                             | Details                                                                                                                                                                                                   |
+| --- | ------------------------- | ------------------------------ | ------- | ------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
The third column `Change` is confusing to me and doesn't seem necessary. I think we can remove it.

Reply:
Removed `Change` column and say it in the `Migration Step` column.